src/gui/Src/BasicView/HexDump.h

```diff
@@ -77,6 +77,7 @@ class HexDump : public AbstractTableView
         int separator = 0;
         QTextCodec* textCodec; //name of the text codec (leave empty if you want to keep your sanity)
         DataDescriptor_t data;
+        std::function<void(QObject *obj)> columnSwitch;

```

Comment:
The `QObject` parameter for `this` is not needed because you can capture `this` (see later comment).